packages/next/client/performance-relayer.ts

```diff
@@ -82,10 +79,21 @@ export default (onPerfEntry?: ReportCallback): void => {
   }
   isRegistered = true
 
-  onCLS(onReport)
-  onFID(onReport)
-  onFCP(onReport)
-  onLCP(onReport)
-  onTTFB(onReport)
-  onINP(onReport)
+  for (const webVital of WEB_VITALS) {
+    const m = config?.attributions.includes(webVital)
+      ? require('next/dist/compiled/web-vitals-attribution')
```

Comment:
It looks like this increases bundle sizes quite a bit `+3.62 kB`, can we add an experimental flag to enable this support and then use an env variable similar to `__NEXT_HAS_REWRITES` to dead-code eliminate the extra `require` when it's not enabled?